frontend/src/container/AppLayout/index.tsx

```diff
@@ -279,6 +278,16 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 
 	const isSideNavCollapsed = getLocalStorageKey(IS_SIDEBAR_COLLAPSED);
 
+	/**
+	 * Note: Right now we don't have a page-level method to pass the sidebar collapse state.
+	 * Since the use case for overriding is not widely needed, we are setting it here
+	 * so that the workspace locked page will have an expanded sidebar regardless of how users
+	 * have set it or what is stored in localStorage. This will not affect the localStorage config.
+	 */
+
+	const isWorkspaceLocked = (pathname: string): boolean =>
```

Comment:
don't need to pass the pathname to the function it's accessible in the component, just make one variable similar to `renderFullScreen` in the same file

Reply:
fixed the code, removed the pathname